Documentation/ceph-object-multisite.md

```diff
@@ -165,12 +166,12 @@ kubectl create -f realm-a-keys.yaml
 
 Once the admin knows the endpoint and the secret for the keys has been created, the admin should create:
 
-1. A [CephObjectRealm](/design/ceph/object/realm.md) matching to the realm on the other Ceph cluster, with an endpoint as described above.
-1. A [CephObjectZoneGroup](/design/ceph/object/zone-group.md) matching the master zone group name or the master CephObjectZoneGroup from the cluster the the realm was pulled from.
-1. A [CephObjectZone](/design/ceph/object/zone.md) referring to the CephObjectZoneGroup created above.
-1. A [CephObjectStore](/design/ceph/object/store.md) referring to the new CephObjectZone resource.
+1. A [CephObjectRealm](https://github.com/rook/rook/blob/{{ branchName }}/design/ceph/object/realm.md) matching to the realm on the other Ceph cluster, with an endpoint as described above.
```

Comment:
Do we really need to reference the design docs, or shall we reference the ceph-object-store-crd.md like above? The design docs are just expected to be for a developer audience, not the admins.